FpCategories/gap/Quivers.gd

```diff
@@ -94,11 +94,15 @@ DeclareAttribute( "QuiverName", IsCapQuiver );
 DeclareAttribute( "NumberOfObjects", IsCapQuiver );
 
 #! @Description
-#!  Returns the labels of objects in the quiver <A>q</A>.
+#!  Returns the labels of objects all in the quiver <A>q</A>.
```

Comment:
you probably mean: `all objects` rather `objects all`

Reply:
done!